build.gradle

```diff
@@ -5,9 +5,9 @@ buildscript {
     repositories {
         mavenCentral()
     }
-//    dependencies {
-//        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
-//    }
+    dependencies {
```

Comment:
There were several commits in this branch which changed how inversion was implemented.
Also, the version was arbitrarily bumped up from 0.4.1 to 0.4.4...? but no artifact was created past 0.4.1 ([see artifact versions here](https://github.com/RocketPartners/artifacts/packages/1757458/versions))

<img width="1261" alt="image" src="https://github.com/user-attachments/assets/503d95ad-d8ed-4487-9e20-308bbf339888">


So, I'm reverting back to 0.4.1 before I added my RDS IAM Auth changes